PBN RCT.Rmd

```diff
@@ -507,6 +522,31 @@ pbn_complete %>%
   group_by(name) %>%
   summarize(mean = mean(value),
             sd = sd(value))
+
+#N and percent of intervention participants with sum score >3, reflect overall perceived SSI acceptability
+included_participants_pfs %>%
+  filter(treatment_group == TRUE) %>%
+  mutate(pi_pfs_sum_scores = pi_pfs_sum / 7) %>%
+  mutate(pi_pfs_sum_scores_acceptable = pi_pfs_sum_scores > 3) %>%
+  summarise(
+    pi_pfs_sum_scores_acceptable_number = sum(pi_pfs_sum_scores_acceptable),
+    pi_pfs_sum_scores_acceptable_percent = (pi_pfs_sum_scores_acceptable_number * 100) / n()
```

Comment:
Not important but feel free to use `scales::percent(mean(pi_pfs_sum_scores_acceptable))` to get the %